README.md

```diff
@@ -108,3 +108,18 @@ The [docker-compose.yml](dist/ci/docker-compose.yml) mentions two container imag
 
 As mentioned before, the main repository uses GitHub Actions to automatically run the tests when a pull request is opened or the code is pushed to the master branch. You can find the details in the
 [workflow definition](.github/workflows/ci-test.yml). Note that, in addition to the steps listed before, code coverage data is submitted to [Codecov](https://app.codecov.io/gh/openSUSE/openSUSE-release-tools).
+
+### Debugging Failures in CI
+
+The first step is to reproduce it when running it locally as described above.
+To see all logs following command can be executed:
+
+  docker-compose -f dist/ci/docker-compose.yml logs -f --tail=10
+
+To check content of any running container use this command ( in this case "api" container ) to get shell:
```

Comment:
```suggestion
You can run commands in any container by using the docker-compose `exec` command. For instance, you can connect to a container through a shell with the following command (in this case, it will connect to the container behind the `api` service):
```